test/python/transpiler/test_csp_layout.py

```diff
@@ -21,13 +21,15 @@
 from qiskit.converters import circuit_to_dag
 from qiskit.test import QiskitTestCase
 from qiskit.providers.fake_provider import FakeTenerife, FakeRueschlikon, FakeTokyo
+from qiskit.utils import optionals
 
 
 class TestCSPLayout(QiskitTestCase):
     """Tests the CSPLayout pass"""
 
     seed = 42
 
+    @unittest.skipUnless(optionals.HAS_CONSTRAINT, "needs python-constraint")
```

Comment:
Can we do the skip at the class level instead of each method? Since these tests will require python constraints because the csp layout class requires the package.

Reply:
Fixed in e3f835b8e.